PDF_UA/2/8.5 Real content without textual semantics/8.5.1 General/verapdf-profile-8-5-1-t01.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:58:04.098+03:00">
+        <name>ISO 14289-2:202x - 8.5 Real content without textual semantics - 8.5.1 General - Figure and Formula</name>
+        <description>Real content that does not possess the semantics of text objects and does not have an alternate 
+            textual representation shall be enclosed within Figure or Formula structure elements as appropriate</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="SESimpleContentItem">
+            <id specification="ISO_14289_2" clause="8.5.1" testNumber="1"/>
+            <description>Real content that does not possess the semantics of text objects and does not have an alternate
+                textual representation shall be enclosed within Figure or Formula structure elements as appropriate</description>
+            <test>itemType == 'text' || parentStructureTag == null || ActualText != null ||
+                isArtifact == true || isFigure == true || isFormula == true</test>
+            <error>
+                <message>Real content that does not possess the semantics of text objects and does not have an alternate
```

Comment:
Real content that does not possess the semantics of text objects and does not have an alternate textual representation is not enclosed within Figure or Formula structure elements